homeassistant/components/solarlog/const.py

```diff
@@ -26,14 +28,15 @@
 class SolarLogSensorEntityDescription(SensorEntityDescription):
     """Describes Solarlog sensor entity."""
 
-    factor: float | None = None
+    value: Callable = round
```

Comment:
Make this a little more specific, define what is expected to be passed into the callable and its return type.

Reply:
Fixed, with some help from @epenet